addons/binding/org.openhab.binding.jablotron/src/main/java/org/openhab/binding/jablotron/handler/JablotronBridgeHandler.java

```diff
@@ -0,0 +1,165 @@
+            logger.error("Cannot start http client!", e);
+            return;
+        }
+        scheduler.execute(this::login);
+    }
+
+    @Override
+    public void dispose() {
+        super.dispose();
+        logout();
+        try {
+            httpClient.stop();
+        } catch (Exception e) {
+            logger.error("Cannot stop http client", e);
+        }
+    }
+
+
+    private void login() {
+        String url = JABLOTRON_URL + "ajax/login.php";
+
+        try {
+            String urlParameters = "login=" + URLEncoder.encode(bridgeConfig.getLogin(), "UTF-8") + "&heslo=" + URLEncoder.encode(bridgeConfig.getPassword(), "UTF-8") + "&aStatus=200&loginType=Login";
+
+            ContentResponse resp = httpClient.newRequest(url)
+                    .method(HttpMethod.POST)
+                    .header(HttpHeader.ACCEPT_LANGUAGE, "cs-CZ")
+                    .header(HttpHeader.ACCEPT_ENCODING, "gzip, deflate")
+                    .header(HttpHeader.REFERER, JABLOTRON_URL)
+                    .header("X-Requested-With", "XMLHttpRequest")
+                    .agent(AGENT)
+                    .content(new StringContentProvider(urlParameters), "application/x-www-form-urlencoded; charset=UTF-8")
+                    .timeout(TIMEOUT, TimeUnit.SECONDS)
+                    .send();
+
+            String line = resp.getContentAsString();
+
+            JablotronLoginResponse response = gson.fromJson(line, JablotronLoginResponse.class);
+            if (!response.isOKStatus()) {
+                logger.error("Invalid response: {}", line);
+                return;
+            }
+            logger.debug("Successfully logged to Jablotron cloud!");
+            updateStatus(ThingStatus.ONLINE);
+        } catch (TimeoutException ex) {
+            updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.COMMUNICATION_ERROR, "Timeout during login to Jablonet cloud");
+            scheduler.schedule(this::login, 30, TimeUnit.SECONDS);
+        } catch (Exception ex) {
+            logger.error("Exception during login to Jablotron cloud: {}", ex.toString());
+            updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.COMMUNICATION_ERROR, "Cannot login to Jablonet cloud");
+        }
+    }
+
+    private void logout() {
+
+        String url = JABLOTRON_URL + "logout";
+        try {
+            ContentResponse resp = httpClient.newRequest(url)
+                    .method(HttpMethod.GET)
+                    .header(HttpHeader.ACCEPT_LANGUAGE, "cs-CZ")
+                    .header(HttpHeader.ACCEPT_ENCODING, "gzip, deflate")
+                    .header(HttpHeader.REFERER, JABLOTRON_URL)
+                    .agent(AGENT)
+                    .timeout(5, TimeUnit.SECONDS)
+                    .send();
+
+            String line = resp.getContentAsString();
+
+            logger.debug("logout... {}", line);
+        } catch (Exception e) {
+            //Silence
+            //logger.error(e.toString());
```

Comment:
remove commented out code. You also almost never want to not log exceptions. Or log to debug or even trace if it's something that should not be a problem. This will always give you the opportunity in case something unclear happens to let a user enable logging and that might show the problem.